src/main/java/uk/gov/hmcts/sptribs/recordlisting/LocationService.java

```diff
@@ -38,8 +39,14 @@ public class LocationService {
     private LocationClient locationClient;
 
     public DynamicList getHearingVenuesByRegion(String regionId) {
-        final var hearingVenues = getCourtVenues(regionId);
-        return populateVenueDynamicList(hearingVenues);
+        final HearingVenue[] hearingVenues = getCourtVenues(regionId);
+        if (hearingVenues == null) {
+            return populateVenueDynamicList(hearingVenues);
```

Comment:
Dont we need to filter based on courtTypeId when hearingVenues is null. Why?

Reply:
when the hearingVenues is null , the filter operation gives error . Also some cases , like region selected as Northern Ireland , but there is no CIC case hearing venue , then the system shows error . that's why I added to avoid the null value from filter operation . Or I can try different approach ?